mlflow/telemetry/events.py

```diff
@@ -176,6 +176,32 @@ class PromptOptimizationEvent(Event):
     name: str = "prompt_optimization"
 
 
+class PromptAdaptationEvent(Event):
+    name: str = "prompt_adaptation"
```

Comment:
```suggestion
    name: str = "adapt_prompt"
```

can we use the same name as the API?